src/main/java/com/thealgorithms/datastructures/hashmap/hashing/MajorityElement.java

```diff
@@ -0,0 +1,26 @@
+import java.util.HashMap;
+
```

Comment:
Add some Description about the Algorithm you are proposing. You could add some links that talks about the problem. like a GFG link mayby